README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+## Jimin Han 101
+My name is **Jimin Han**. \
+I am from Korea. *Yes, South not North*.\
```

Comment:
If you want line breaks, you can also just leave a blank line between each sentence.

```
My name is **Jimin Han**.

I am from Korea. *Yes, South not North*.
```

My name is **Jimin Han**.

I am from Korea. *Yes, South not North*.